docs/getting_started.md

```diff
@@ -12,6 +12,9 @@
    Latest client can be downloaded from [oc-client](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
 4. For vSphere based installations, [terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
    and [jq]( https://stedolan.github.io/jq/download/) should be installed ( terraform version should be 0.11.13  )
+5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
+   After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
+   parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
```

Comment:
Can the clone and setting up the path to ui-testing.yaml not be done in PR?

Reply:
I wanted to avoid this as if someone will run locally UI tests they usually would like to develop/debug UI testing and would like to point to their own location of the openshitc console repo they are currently working on.  As user will usually have some changes in this repo it would mean that as part of setup I will change to some branch and so on and this will not be intentional as it will delete the users changes.   We can discuss on automation mtg if we would like to have different approach but I wouldn't change logic in this PR as this currently working the same approach as we already using in jenkins for UI deployment so no need for any change in jenkins jobs.

Reply:
Sure we can discuss more in the call, the point is to avoid additions steps to run UI test. We can take this as follow on PR as well.